diff --git a/setup.py b/setup.py
index 0dcdf66..104bae8 100755
--- a/setup.py
+++ b/setup.py
@@ -50,7 +50,7 @@ if build_ext:
                      extra_compile_args=cargs,
                      swig_opts=['-keyword', '-nodefaultctor',
                                 '-nodefaultdtor', '-noproxy'],
-                     optional=True,
+                     optional=False,
                      **ext_limited_args)]
 else:
     mod = []
